backend/h4tcsacapi/settings.py

```diff
@@ -59,9 +58,9 @@
     'django.contrib.staticfiles',
     'rest_framework',
     'h4tcsacapp',
+    'h4tcsacadmin',
     'corsheaders',
-    'h4tcsacrecertify'
-    'h4tcsacadmin'
+    'django_extensions'
```

Comment:
This is an app that allows you to check a lot of Django configs that you can't directly see.